techniques/systemSettings/userManagement/userManagement/8.0/metadata.xml

```diff
@@ -0,0 +1,191 @@
+<!--
+Copyright 2016 Normation SAS
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, Version 3.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<TECHNIQUE name="Users">
+  <DESCRIPTION>This technique manages the target host(s) users.
+  It will ensure that the defined users are present on the system.</DESCRIPTION>
+  <MULTIINSTANCE>true</MULTIINSTANCE>
```

Comment:
We should not remove compatible OS information.

Reply:
Still there